index.php

```diff
@@ -46,24 +48,42 @@
     $uri = '/';
 }
 
-$siteFound = false;
-if(doesSiteExist('portal', $sitePath))//query for portal
-{
-    $db_config = new DB_Config($sitePath);
-    $config = new Config($sitePath);
-    $leafRoutes = new LEAFRoutes('portal');
-    $siteFound = true;
-}elseif(doesSiteExist('nexus', $sitePath))//query for nexus
-{
-    $config = new Orgchart\Config($sitePath);
-    $leafRoutes = new LEAFRoutes('nexus');
-    $siteFound = true;
+//skip config lookup if looking for static files
+$cssPos = strpos($uri, '/css/');
+$jsPos = strpos($uri, '/js/');
+if ($cssPos !== false || $jsPos !== false) {
+    $orgchartPos = strpos($uri, '/orgchart/');
```

Comment:
does the first character in `/orgchart/` serach for literal /?

Should it be 'orgchat/' ?

Reply:
Does case sensitivity matter?

What about following site:
<REDACTED>
![image](https://user-images.githubusercontent.com/59398225/97449687-54519e80-1908-11eb-86ea-fbcb8580f25e.png)

Reply:
resolved